cmd/api-response.go

```diff
@@ -89,7 +89,8 @@ type ListVersionsResponse struct {
 	IsTruncated bool
 
 	CommonPrefixes []CommonPrefix
-	Versions       []ObjectVersion
+	DeleteMarkers  []DeleteMarkerVersion `xml:"DeleteMarker,omitempty"`
+	Versions       []ObjectVersion       `xml:"Version,omitempty"`
```

Comment:
```suggestion
	Versions       []ObjectVersion       `xml:"Versions,omitempty"`
```

Reply:
This comment can be ignored. I had missed the point that xml's naming convention w.r.t tag names for array elements are different from json.